lib/r10k/cli/deploy.rb

```diff
@@ -60,6 +60,13 @@ def self.command
               exit 1
             end
           end
+          option nil, :'puppet-conf', 'Path to puppet.conf', argument: :required do |value, cmd|
+            unless File.exist? value
```

Comment:
Should we check readability instead of existence?

Reply:
Switched the `puppet-conf` checks for `exist?` to `readable?`.